src/code/actions.js

```diff
@@ -268,16 +268,20 @@ function getEarnedCoinString(state) {
   return ["~ALERT.NEW_PIECE_OF_COIN", scoreString];
 }
 
+function getVisibleGenes(state, routeSpec) {
+  const visibleGenes = AuthoringUtils.getChallengeDefinition(state.authoring, routeSpec).visibleGenes;
```

Comment:
Should the `userChangeableGenes` be added to this list? For all the trials involving the user changing pulldowns, this will result in sending `editableGenes: null` in the `submitDrake` action (or if the authors add an unchangable gene label, it will send just that label). I feel like this can't be what is wanted.

In the basket challenges, clearly we just want the visible genes. So I could see two options: (1) merge the two arrays, to submit all genes the user can see, or (2) send the userChangeableGenes if there are any, and the visibleGenes if there aren't.